int32 order
int64 traj_id
time start_time
#geometry_msgs/Point start_pt

float64[] knots
geometry_msgs/Point[] pos_pts

float64[] yaw_pts
float64 yaw_dt
